.github/mergify.yml

```diff
@@ -0,0 +1,42 @@
+pull_request_rules:
+  - name: Approve and merge non-major version dependabot upgrades
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
```

Comment:
Should just be `author=dependabot[bot]`?

Dependabot preview is shut down now:
https://github.blog/2021-04-29-goodbye-dependabot-preview-hello-dependabot/

Reply:
didn't know about this - will do! (same as above, should also fix upstream)